shop/src/main/java/ro/msg/learning/shop/domain/Customer.java

```diff
@@ -0,0 +1,48 @@
+package ro.msg.learning.shop.domain;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.antlr.v4.runtime.misc.NotNull;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "customer")
+@AllArgsConstructor
+@Builder
+public class Customer {
+
+    @Id
+    @Column(name = "id")
+    @UuidGenerator
+    private UUID customerId;
+
+    @NotNull
+    @Column(name = "firstname")
```

Comment:
Let's use this `@Column(...)`  only if we need it. If the column has the same name as in the database it will be linked automatically.

Reply:
This is not really resolved. It can be omitted now because we have little time to implement this app but take this into consideration for the future please!